org.eclipse.scout.rt.jackson.test/src/test/java/org/eclipse/scout/rt/jackson/dataobject/JsonDataObjectsSerializationTest.java

```diff
@@ -3806,41 +3806,66 @@ public void testSerializeDeserializeNestedRawDoTypes() throws Exception {
     String inputJson = readResourceAsString("TestNestedRawDo.json");
     TestNestedRawDo testDo = s_dataObjectMapper.readValue(inputJson, TestNestedRawDo.class);
 
+    DoEntity entity2 = BEANS.get(DoEntity.class);
+    // NOTE: This test-case is not a supposed real-world example. An IId as part of an untyped DoEntity is serialized unqualified and may never be deserialized correctly.
```

Comment:
DoEntity ws. DoEntityBuilder below, use one or the other, but not both?

Reply:
Used both on purpose: DoEntityBuilder returns IDoEntity, therefore for the DoValue<DoEntity> I need the  BEANS.get(DoEntity.class) directly.